doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -1,223 +1,275 @@
+.When you try to invoke a Business Knowledge Model (BKM) function from included DMN model, the validation fails [https://issues.redhat.com/browse/RHDM-1395[RHDM-1395]]
 
-Issue: When editing a case definition in process designer, you can add case file variables in the *Case Management* section, but you cannot delete them. It is also not possible to delete the global variables.
+Issue: You can not call the BKM function imported from another DMN model, which is aliased with the name containing `.` characters. If you try to invoke that function, the validation fails.
 
 Steps to reproduce:
 
-. Create a case project.
-. Create a case definition.
-. In the *Properties* panel add a case file variable in the *Case Management* section.
-. Add a global variable in the *Properties* panel.
-. Try to delete the case file variable and global variable.
+* Create an empty `Salary.dmn` DMN model.
+* Create an another empty `WorkingHours.dmn` DMN model.
+* Import `WorkingHours.dmn` DMN model into `Salary.dmn` DMN model.
+* Set an alias for `WorkingHours.dmn` model by including `.` character. For example, `working.hours.model`.
+* Validate the model. You receive an error message.
+
+Workaround: None
+
+== {PLANNER}
+
+.In Constraint Streams API, the performance of incremental score calculations is degraded using `groupBy()` method [https://issues.redhat.com/browse/RHDM-1385[RHDM-1385]]
 
-Expected result: A trash can icon appears next to each variable. After you click the icon, the variable is removed.
+Issue: When you use the `groupBy()` method in Constraint Streams API, the the performance of incremental score calculations is degraded on high level.
 
-Actual result: It is not possible to delete variables and the trash can icon is missing.
+Workaround: You can use the DRL score calculations.
 
-Workaround:
+== {OPENSHIFT}
 
-. Download the process or a case.
-. Locate the case file variable and global variable in the downloaded `.bpmn` file and delete them.
-. Delete the process or a case from {CENTRAL}.
-. Import the previously downloaded and edited `.bpmn` file back into {CENTRAL}.
+ifdef::PAM[]
 
-.Ruleflow group is not populated [https://issues.redhat.com/browse/RHPAM-2740[RHPAM-2740]]
+.You can not create custom database image for PostgreSQL using PostgreSQL enterprise instance [https://issues.redhat.com/browse/RHPAM-3005[RHPAM-3005]]
 
-Issue: In the process designer, the Ruleflow group menu is not populated with the rule flow groups defined in the project.
+Issue: When you create a custom database for PostgreSQL and push the image using PostgreSQL enterprise instance, {KIE_SERVER} fails to start.
 
 Steps to reproduce:
 
-. Create a new DRL file containing a rule flow group.
-. Create a new process.
-. Activate the Business Rule task.
-. Click the *Rule Flow Group* property of the Business Rule task.
+* Create a custom database and make the image build for PostgreSQL.
```

Comment:
I'd suggest using a numbered list instead of bullets since these are sequential steps.